http3/response_writer.go

```diff
@@ -70,6 +70,7 @@ type responseWriter struct {
 	headerWritten bool
 	contentLen    int64 // if handler set valid Content-Length header
 	numWritten    int64 // bytes written
+	isHead        bool
```

Comment:
Nit: if you move `headerWritten` down here, this struct will be 8 bytes smaller (I think).

Reply:
Good point out. I have made this change.